src/Microsoft.ML.Data/Model/ModelOperationsCatalog.cs

```diff
@@ -187,6 +201,13 @@ public ITransformer Load(Stream stream, out DataViewSchema inputSchema)
         /// <param name="inputSchema">Will contain the input schema for the model. If the model was saved without
         /// any description of the input, there will be no input schema. In this case this can be <see langword="null"/>.</param>
         /// <returns>The loaded model.</returns>
+        /// <example>
+        /// <format type="text/markdown">
+        /// <![CDATA[
+        /// [!code-csharp[Load](~/../docs/samples/docs/samples/Microsoft.ML.Samples/Dynamic/Transforms/CustomMappingSaveAndLoad.cs)]
```

Comment:
>CustomMappingSaveAndLoad [](start = 103, length = 24)

I think majority of the Load()/Save() operations don't require mlContext.ComponentCatalog.RegisterAssembly that CustomMapping requires, which makes this sample the exception not the rule. If that's assumption is correct, let's stick to a sample that doesn't scare users with using  RegisterAssembly, CustomMappingFactoryAttribute, and Action<>. You can use the same approach I suggested for CreatePredictionEngine sample below (simple data, simple pipeline) and just simple Save() then Load().

CustomMappingSaveAndLoad is best suited for CustomMapping API which we already use it there.

Reply:
Agree for this one, CustomMapping might distract them. Maybe a simpler pipeline. 

---
In reply to: [275778414](https://github.com/dotnet/machinelearning/pull/3352#discussion_r275778414) [](ancestors = 275778414)